File: Electronics_Shop_17.Services/Database/UserNotification.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Electronics_Shop_17.Services.Database
{
    public class UserNotification : IHelperId
    {
        public int Id { get; set; }
        public int UserAccountId { get; set; }
        public int NotificationId { get; set; }
        public bool IsRead { get; set; }

    }
}

File: angular17_web/src/app/models/user-notification/user.notification.ts
export class UserNotification {
  id!: number;
  userAccountId!: number;
  notificationId!: number;
  isRead!: boolean;

  constructor(data?: any) {
    if (!data) return;

    this.id = data.id;
    this.userAccountId = data.userAccountId;
    this.notificationId = data.notificationId;
    this.isRead = data.isRead;
  }
}
